src/constants.js

```diff
@@ -95,9 +95,13 @@ export const FEATURE_VECTORS_TAB = 'feature-vectors'
 export const ADD_TO_FEATURE_VECTOR_TAB = 'add-to-feature-vector'
 
 export const DATASETS_PAGE = 'DATASETS'
+export const DATASETS_TAB = 'datasets'
```

Comment:
We have `DATASETS` constant that has identical value `datasets`. That's means you can replace `DATASETS` with `DATSETS_TAB`
Also, you can replace `files` with `FILES_TAB` in `parseUri.js` and `ProjectMonitor.js` files